website/content/en/docs/contribution-guidelines/releasing.md

```diff
@@ -160,35 +154,54 @@ git pull
 git checkout -b release-$RELEASE_VERSION
 ```
 
-Run the pre-release `make` target:
+
+#### 2. Prepare the release commit
+
+1. Update the top-level [Makefile] variable `IMAGE_VERSION` to the upcoming release tag `v1.3.1`.  This variable ensures sample projects have been tagged
+correctly prior to the release commit.
+
+  ```sh
+  sed -i -E 's/(IMAGE_VERSION = ).+/\1v1\.3\.1/g' Makefile
+  ```
+
+1. Run the pre-release `make` target:
 
 ```sh
 make prerelease
 ```
 
-The following changes should be present:
+1. Regenerate testdata. 
 
-- `changelog/generated/v1.3.0.md`: commit changes (created by changelog generation).
-- `changelog/fragments/*`: commit deleted fragment files (deleted by changelog generation).
+```sh
+make test-sanity
+```
 
-Commit these changes and push:
+All of the following changes should be present (and no others).
+
+1. `Makefile`: IMAGE_VERSION modified
+1. changelog/: 
+  1. all fragments deleted
+  1. changelog/generated/v1.3.1.md
+1. docs: 
+    add: `website/content/en/docs/upgrading-sdk-version/v1.3.1.md` (only if there are migration steps)
+    modified: installation docs link update. Is this still broken?TODO(asmacdo)
```

Comment:
Nope all fixed (although the script is broken and [removes the `/download` path element](https://github.com/operator-framework/operator-sdk/blob/master/website/scripts/update_download_url.sh#L13), feel free to fix that in this PR).
```suggestion
    modified: installation docs link update.
```